README.md

```diff
@@ -95,15 +95,15 @@ For a complete working example, check out the sample apps included in this repo.
 
 #### Swift
 ```swift
-func presentCropViewController {
+func presentCropViewController() {
   let image: UIImage = ... //Load an image
   
-  let cropViewController = CropViewController(image: image)
+  let cropViewController = TOCropViewController(image: image)
```

Comment:
This isn't necessary. `CropViewController` is the name of the Swift wrapper around the Objective-C version. :)

```suggestion
  let cropViewController = CropViewController(image: image)
```